.github/workflows/mumuk-ci-cd.yml

```
@@ -69,25 +73,11 @@ jobs:
-              sudo docker ps -qa | xargs -r sudo docker rm -f
-            fi
-            
-            IMAGES=$(sudo docker images -q)
-            if [ -n "$IMAGES" ]; then
-            sudo docker rmi $IMAGES
-            fi
-  
-            sudo docker-compose up --build -d
-            sudo docker image prune -f
+            sudo docker-compose up -d --build
+            sudo docker image prune -f
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)